client/src/pages/recording-view/components/plugins-pane.tsx

```diff
@@ -305,7 +335,7 @@ export const PluginsPane = () => {
           ))}
         </select>
       </label>
-      {connectionInfo && (
+      {dataSource && (
```

Comment:
We should probably do a different check here for exemole you don't want that for local files but you do want this for personal blob storage